vmdb/app/models/configured_system.rb

```diff
@@ -12,6 +12,10 @@ class ConfiguredSystem < ActiveRecord::Base
 
   alias_method :manager, :configuration_manager
 
+  def self.common_configuration_profiles_for_selected_hosts(ids)
```

Comment:
@AparnaKarve As discussed, In the case that the user selects multiple ConfiguredSystems that don't all have something in common, they would enter the dialog and have nothing to choose for ConfigurationProfile.  I think it would be nice to let them know before entering the dialog that their selection can not be provisioned.  Passing the selected id's into this method will return nil if they have nothing in common.